apply.js

```diff
@@ -137,8 +16,37 @@ const allowedHostnames = [
   "eduvulcan.pl",
 ];
 
+/** @returns {Promise<Patch[]>} */
+const fetchPatches = async () => {
+  const patchesResponse = await fetch(chrome.runtime.getURL("patches.json"))
+  return JSON.parse(await patchesResponse.text());
+}
+
+const getConfig = async () => (await chrome.storage.sync.get("options"))?.options ?? {};
+
+chrome.storage.onChanged.addListener((changes, namespace) => {
+  if (!changes.options?.oldValue || changes.options?.oldValue === changes.options?.newValue) return
+  if (namespace !== "sync") return;
+  window.location.reload()
+})
+
+const getPatchesFiles = (patches, config) => {
+  return patches.flatMap((patch) => {
+    const result = []
+    if (config[patch.name].enable) {
+      if (patch.files?.js?.length) result.push(patch.files.js.map((file) => `patches/${file}`));
+      if ((!patch.allowedHostsCss || patch.allowedHostsCss.includes(window.location.hostname)) && patch.files?.css?.length) {
+        result.push(patch.files.css.map((file) => `patches/${file}`))
+      }
+    }
```

Comment:
```suggestion
    if (!config[patch.name].enable) return;
    if (patch.files?.js?.length) result.push(patch.files.js.map((file) => `patches/${file}`));
    if ((!patch.allowedHostsCss || patch.allowedHostsCss.includes(window.location.hostname)) && patch.files?.css?.length) {
        result.push(patch.files.css.map((file) => `patches/${file}`))
    }
```

Reply:
Your code suggestion even introduces an error, there is `return result;` after that for a reason and it's as readable as this